crates/rbuilder/src/live_builder/mod.rs

```diff
@@ -202,9 +202,9 @@ where
                 continue;
             }
             // Allow only increasing blocks
-            if last_processed_block.map_or(false, |last_processed_block| {
```

Comment:
Is this change required?

Reply:
Yeah, `make lint` kept failing if I didn't address this... Probably due to Clippy (with -D warnings) treating warnings as errors